modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/servlet/taglib/ManagementToolbarTag.java

```diff
@@ -1250,12 +1251,14 @@ protected int processStartTag() throws Exception {
 
 			jspWriter.write(
 				LanguageUtil.format(
-					resourceBundle, "x-results-for-x",
-					new Object[] {
-						getItemsTotal(),
-						(searchValue == null) ? StringPool.BLANK :
-							HtmlUtil.escape(searchValue)
-					}));
+					resourceBundle, "x-results-for",
+					new Object[] {getItemsTotal()}));
+
+			jspWriter.write("<strong>");
+			jspWriter.write(
+				HtmlUtil.escape(
+					StringBundler.concat(" ", '"', searchValue, '"')));
```

Comment:
Hey @julien! You are right but when I execute `gw formatSource` with `StringPool.SPACE` it fails, that's why I put `" "` and `" "`

Reply:
@sergiojimcos thanks for the information. I suggest we have a look at the error together just to see if it's something we can fix. If you use `git grep -w StringPool\.SPACE` you'll see that it's used in various places and when I see the results; I'm suspecting that `StringPool.SPACE` should be on its own line, i.e.

Instead of 

```java
StringBundler.concat(" ", '"', searchValue, '"')));
```

We might have to use 

```java
StringBundler.concat(StringPool.SPACE);
StringBundler.concat(StringPool.QUOTE);
StringBundler.concat(searchValue);
StringBundler.concat(StringPool.QUOTE);
```

In any case let's analyze the error message and see if it gives us a hint (most warning or error messages generated by the source formater often have a link to describe how to fix the issue).

Reply:
@julien i've been tested what you have suggested me, the only problem is `StringBundler.concat()`. When I execute `gw formatSource`, there is a warning massage that specifies `StringBundler` should not be used when concatenating less than 3 elements. Finally, I have followed what you have suggested replacing the `StringBundler.concat` to `HtmlUtil.escape()`.

Reply:
> Finally, I have followed what you have suggested replacing the StringBundler.concat to HtmlUtil.escape().

@sergiojimcos let's have a call about this because I don't understand this part